index.js

```diff
@@ -0,0 +1,7 @@
+function divide (x, y) {
+    return x / y;
+}
```

Comment:
Using an arrow function:
```suggestion
const divide = (x, y) => x / y;
```